internal/expr/expr_test.go

```diff
@@ -66,8 +84,8 @@ var tests = []struct {
 	typeSamples:    []any{Person{}},
 	expectedSQL:    "SELECT p.address_id AS _sqlair_0, p.id AS _sqlair_1, p.name AS _sqlair_2",
 }, {
-	summary:        "spaces and tabs",
-	query:          "SELECT p.* 	AS 		   &Person.*",
+	summary: "spaces and tabs",
+	query: "SELECT p.* 	AS 		   &Person.*",
```

Comment:
Not sure what happened here but I think this looks unformatted?

Reply:
This is my number one reason for switch to Goland. No idea what is wrong with my vim setup, I've tried changing gofmt versions, but it doesn't seem to be using the one on my path.